main/src/library/File.flix

```diff
@@ -15,6 +15,176 @@
  */
 namespace File {
 
+    ///
+    /// File types.
+    ///
+    enum FileType with Eq, Order, ToString {
+        case File,
+        case Directory,
+        case SymLink,
+        case Other
+    }
+
+    ///
+    /// Returns the access time of the given file `f` in milliseconds.
+    ///
+    pub def accessTime(f: String): Result[Int64, String] & Impure =
+        try {
+            import new java.io.File(String) as newFile;
+            import java.io.File.toPath();
+            import static java.nio.file.Files.readAttributes(##java.nio.file.Path, ##java.lang.Class, Array[##java.nio.file.LinkOption]);
+            import static java.lang.Class.forName(String);
+            import java.nio.file.attribute.BasicFileAttributes.lastAccessTime();
+            import java.nio.file.attribute.FileTime.toMillis();
+
+            let javaFile = newFile(f);
+            let javaPath = toPath(javaFile);
+            let classOfAttributes = forName("java.nio.file.attribute.BasicFileAttributes");
+            let fileAttributes = readAttributes(javaPath, classOfAttributes, []);
+            let timeLastAccess = lastAccessTime(fileAttributes);
+            let inMilliSec = toMillis(timeLastAccess);
```

Comment:
`timeLastAccess` is of type [FileTime](https://docs.oracle.com/javase/7/docs/api/java/nio/file/attribute/FileTime.html).
Just if we want a date object instead, it should be possible to convert it to something other than milliseconds.